### ISSUE TITLE
the model trained by SDG optimizer is too bad

### ISSUE THREAD

**Issue by chenxu93** [michuanhaohao/reid-strong-baseline#64]
I want to know why the mAp is about 15 percentage worse when use SGD optimizer, how does set the parameter for SGD? The mAP result obtined by default Adam optimizer is match with the paper report. Do you study the parameters for SGD optimizer?

**Comment by michuanhaohao** [comment]
@chenxu93 Did you use SGD with warmup? I did not try SGD.

**Comment by he010103** [comment]
@chenxu93 Have you set the appropriate learning rate？

**Comment by L1aoXingyu** [comment]
@chenxu93 If you use SGD, you need to set a larger lr than Adam. But we don't recommend to use SGD, because it's a little bit tricky for tuning lr.

**Comment by chenxu93** [comment]
@michuanhaohao @L1aoXingyu Thanks, maybe the SGD is more useful for the detection task, and Adam is more suitable for the classify task.
